### PR TITLE
Documentation: Update documentation about android and tizen web application

### PR DESCRIPTION
**Pull request by niley7464**
This PR updates documentation about android and tizen web application.

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: android/kotlin_app/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+We assume that you already have experienced Android application developments with Android Studio.
+
+ * Host PC:
+   * OS: Ubuntu 20.04 / 22.04 x86_64 LTS
+   * Android Studio: Ubuntu version
+ * Target Device:
+   * CPU Architecture: ARM 64bit (aarch64)
+   * Android SDK: Min version 34
+   * [nnstreamer-android application](https://github.com/nnstreamer/nnstreamer-android/blob/main/documentation/getting_started.md#building-the-android-app-with-llama2) is installed.
```
Comment (niley7464) [inline android/kotlin_app/README.md:13]: https://github.com/nnstreamer/nnstreamer-android/pull/117
is revewing.

File: android/README.md [hunk 2]
```diff
@@ -7,3 +7,8 @@ This guide is suitable for normal application developers.
 
 * __Android Platform Guide__: If you are going to build Android rootFS from scratch for NNStreamer, then refer to [this guide](https://github.com/nnstreamer/nnstreamer-example/tree/master/android/android_platform_guide.md).
 This guide is _only_ suitable for Android platform developers or system guys.
+
+Also, we provide a [single APK integration](https://github.com/nnstreamer/nnstreamer-android) of NNStreamer and ML API.
+This application is a solution for on-device Machine Learning Operations (MLOps) and provides end-points to ML task.
+
+* __Android App using MLOps Solution__: If you want to make an Android Application without dependencies such as `aar`, then refer to [this guide](/android/kotlin_app/README.md).
```
Comment (wooksong) [inline android/README.md:14]:
```suggestion
Also, we provide a [single APK integration](https://github.com/nnstreamer/nnstreamer-android) of NNStreamer and ML API.
This application is a solution for on-device Machine Learning Operations (MLOps) and provides end-points to ML tasks.

* __Android App using MLOps Solution__: If you want to make an Android Application without dependencies such as `aar`, refer to [this guide](/android/kotlin_app/README.md).
```

File: android/kotlin_app/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Build example
+
+We provide Kotlin example which runs an LLM task.
```
Comment (wooksong) [inline android/kotlin_app/README.md:17]:
```suggestion
We provide a Kotlin example that runs an LLM task.
```

File: android/kotlin_app/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+#### Cloning the GitHub Repository
+
+To build example application, you need to clone the repository.
```
Comment (wooksong) [inline android/kotlin_app/README.md:21]:
```suggestion
To build the example application, you need to clone the repository.
```

**Review by wooksong — APPROVED** [review]
LGTM

File: android/README.md (hunk 2 above)
Comment (niley7464) [reply android/README.md]: Updated All!
Thank you 👍

**Pull request merged**